checks.d/redisdb.py

```diff
@@ -215,6 +218,16 @@ def _check_db(self, instance, custom_tags=None):
                             self.warning("{0} key not found in redis".format(key))
                         self.gauge('redis.key.length', 0, tags=key_tags)
 
+        # Save the replication delay for each slave
+        for key in info.keys():
```

Comment:
you shouldn't use .keys () as it would create a copy of the keys.

You should just do:
```python
for key in info
```